src/components/LabeledTags.vue

```diff
@@ -48,10 +48,10 @@ export default defineComponent({
 .labeled-tag {
   background-color: $color-alto;
   border: 1px solid $color-white;
-  border-radius: 10px;
-  margin-right: 0.25rem;
+  border-radius: 5px;
+  margin-right: 0.12rem;
   max-width: 90%;
-  padding: 0.05rem 0.5rem;
+  padding: 0.3rem 0.6rem;
```

Comment:
I know this was the requested value, but it is translated to `4.8px 9.6px` - Instead of dealing with fractional pixels I'd rather we round the given values and use them as such

```suggestion
  padding: 5px 10px;
```

Reply:
Done